src/app/zip-file-explorer/zip/zip.component.scss

```diff
@@ -333,33 +353,33 @@ textarea{
   overflow-x: auto;
   white-space: pre;
   background-color: $Saturation100;
-  color: rgb(161, 229, 238);
+  color: rgb(73, 156, 233);
   cursor: default;
 
 }
 //header css
 .header {
- 
-  max-width: 100%;
-  background-color: #ff6300;
   /*
   *background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0)60%, rgba(255, 77, 246, 0.24));
-  */padding-left:10em;
-  padding-top: 2px;
-  height: 30px;
-  min-height: 30px;
-  margin: 0%;
+  */
   box-shadow: $Shadow;
+  display: flex;
+  justify-content: center;
+  font-size: 4vw;
 }
 .header > span{
   // text-shadow: 1px 0px 1px #faea07f3, -1px 0px 1px #f70cd7f3;
   font-size: 1.5em;
   color: $TextImportant;
 }
-.display-text{
+::ng-deep .display-text{
   color: $TextImportant;
   background-color: $Saturation80;
   padding: 5px 0px 5px 0px;
-  word-wrap: normal;
+  font-size: 1.35vw;
+  max-width: 13vw;
+  max-height: 2.5vh;
+  margin-right: 0px;
 }
+
 //end of code display css
```

Comment:
If we are doing //begin, //end type structure, it should be refactored to leverage region & endregion.

Additionally, requires terminating new line.